packages/icons/scripts/build.js

```diff
@@ -21,6 +21,15 @@ const svgo = new SVGO({
   ]
 });
 const buildDirPath = path.join(__dirname, "../", "dist");
+const distDirPath = path.join(__dirname, "../../../", "dist", "packages", "icons", "dist");
+
+const mkdirSyncRecursive = (directory) => {
```

Comment:
Would `mkdirSync(path, {recursive: true})` do the trick?
https://nodejs.org/api/fs.html#fs_fs_mkdirsync_path_options

Reply:
Sadly not. I spent a while trying that and it didn't work. I'm on Node 10.14.2